sources/packages/web/src/components/generic/HeaderNavigator.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <slot name="title">
+    <div v-if="!routeName" class="form-header-title">{{ title }}</div>
+    <div v-if="routeName" class="form-header-title">
+      <a @click="goBack(routeName, routeParams)">
+        <v-icon left> mdi-arrow-left </v-icon> {{ title }}</a
+      >
+    </div>
+  </slot>
+  <v-row>
+    <v-col>
+      <slot name="subTitle">
+        <div class="form-header-sub-title">
+          {{ subTitle }}
+        </div>
+      </slot>
+    </v-col>
+    <v-col>
+      <div class="float-right ml-2 form-header-button">
+        <slot name="buttons"> </slot>
+      </div>
+    </v-col>
+  </v-row>
+</template>
+<script lang="ts">
+import { useRouter } from "vue-router";
+
+export default {
+  props: {
+    title: {
+      type: String,
+    },
+
+    subTitle: {
+      type: String,
+    },
+    routeName: {
```

Comment:
Instead of using two parameters, can the component just received the same parameter for router.push, as shown below?
![image](https://user-images.githubusercontent.com/61259237/152858021-1b54b018-ae19-4dc8-aa3a-0ddbd956aaf9.png)

Reply:
Yes. Precisely one attribute is sufficient here.